src/_includes/base.njk

```diff
@@ -4,7 +4,7 @@
   	{% block head %}
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<title>{{ title }}</title>
+	<title>{{ title or "NO TITLE" }}</title>
```

Comment:
This was seemingly causing errors due to `throwOnUndefined: true`. So one of your templates might be missing a title. :shrug: